docs/upgrading.asciidoc

```diff
@@ -27,3 +27,11 @@ We love all our products, but sometimes we must say goodbye to a release so that
 forward on future development and innovation.
 Our https://www.elastic.co/support/eol[End of life policy] defines how long a given release is considered supported,
 as well as how long a release is considered still in active development or maintenance.
+The table below is a simplified description of this policy.
+
+[options="header"]
+|====
+|Agent version |EOL Date |Maintained until
+|1.x.x  |2024-01-20 |2.x.x
```

Comment:
I think "maintained until" should be `2.0.0`.

Reply:
thanks